src/bool.js

```diff
@@ -1,5 +1,5 @@
 import { Int } from "./int";
-import { isBoolean } from "lodash";
+import isBoolean from "lodash/isBoolean";
```

Comment:
it's a common pattern to import lodash as `_` and then call its function as `_.isBoolean` for example.